### PR TITLE
Scheduler: Add new `observation` event.

### PR DESCRIPTION
**Pull request by tribeiro**

### CODE REVIEW

File: sal_interfaces/Scheduler/Scheduler_Events.xml [hunk 1]
```diff
@@ -973,4 +973,74 @@
...
+    <item>
+      <EFDB_Name>targetId</EFDB_Name>
+      <Description>Unique target id.</Description>
+      <IDL_Type>long</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>ra</EFDB_Name>
+      <Description>Right Ascention of the target.</Description>
```
Comment (mareuter) [inline sal_interfaces/Scheduler/Scheduler_Events.xml:989]: Ascention --> Ascension

File: sal_interfaces/Scheduler/Scheduler_Events.xml [hunk 2]
```diff
@@ -973,4 +973,74 @@
...
+    <item>
+      <EFDB_Name>decl</EFDB_Name>
+      <Description>Declination of the target.</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>deg</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>rotSkyPos</EFDB_Name>
+      <Description>The rotation angle of the camera relative to the sky E of N (Radians).</Description>
```
Comment (mareuter) [inline sal_interfaces/Scheduler/Scheduler_Events.xml:1003]: This should be degrees.

File: sal_interfaces/Scheduler/Scheduler_Events.xml [hunk 3]
```diff
@@ -973,4 +973,74 @@
...
+      <IDL_Type>double</IDL_Type>
+      <Units>second</Units>
```
Comment (mareuter) [inline sal_interfaces/Scheduler/Scheduler_Events.xml:1005]: Shouldn't this be `deg`?

File: sal_interfaces/Scheduler/Scheduler_Events.xml [hunk 4]
```diff
@@ -973,4 +973,74 @@
...
+    <item>
+      <EFDB_Name>nexp</EFDB_Name>
+      <Description>Number of exposures.</Description>
+      <IDL_Type>long</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>additionalInformation</EFDB_Name>
+      <Description>Any additional information produced by the Scheduler or scheduling algorith in a yaml format.</Description>
```
Comment (mareuter) [inline sal_interfaces/Scheduler/Scheduler_Events.xml:1039]: algorith --> algorithm

**Review by mareuter — APPROVED** [review]
A few things to look at on this, but otherwise OK.

**Review by rbovill — APPROVED** [review]
Changes approved.

**Pull request merged**